ui/shared/components/page/PageHeader.jsx

```diff
@@ -41,12 +41,14 @@ const NO_HEADER_PAGE_TITLES = {
 }
 
 const SIMPLE_HEADER_PAGES = [
-  { page: 'data_management', pages: DATA_MANAGEMENT_PAGES },
+  { page: 'data_management', component: DataManagementPageHeader },
   { page: 'report', pages: REPORT_PAGES },
-].map(({ page, ...props }) => ({
+].map(({ page, component, ...props }) => ({
   key: page,
   path: `/${page}/:subPage?`,
-  component: ({ match }) => <SimplePageHeader page={page} subPage={match.params.subPage} {...props} />,
+  component: ({ match }) => React.createElement(
+    component || SimplePageHeader, { page, subPage: match.params.subPage, ...props },
```

Comment:
Is adding a `SimplePageHeader` component entry for the report page configuration better?

Reply:
no I started that way, but because there is a lot of route adding and handling thats shared it ended up being a lot of copy/paste